toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -41,6 +41,8 @@ type SystemConfig struct {
 	PreserveTdnfCache    bool                      `json:"PreserveTdnfCache"`
 	ReadOnlyVerityRoot   ReadOnlyVerityRoot        `json:"ReadOnlyVerityRoot"`
 	EnableHidepid        bool                      `json:"EnableHidepid"`
+	DisableDocs          bool                      `json:"DisableDocs"`
+	DisableLocales       bool                      `json:"DisableLocales"`
```

Comment:
Since we're already working on locales/languages: what do you think about making this option reflect RPM's configuration and accept a list of languages instead of boolean (internally it would need to be converted to a colon-separated string)? This way we're not unnecessarily limiting ourselves and the users.

Reply:
You can now just set an override as a string, I figure its up to the user to write the string.